sprimber-examples/sprimber-webui-template/src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+logging:
+  level:
+    com.griddynamics.qa.sprimber.lifecycle.TestCaseIlluminator: DEBUG
+    com.griddynamics.qa.sprimber.engine.executor: DEBUG
+sprimber:
+  configuration:
+    featurePath: feature/**/*.feature
+    summary:
+      printer:
+        enable: true
+    tagFilters:
+      - "@smoke or @navigation or @getInTouch"
+web-driver:
```

Comment:
please make the additional `application-windows.yml` configuration file, so we may have one profile for windows based machine, one for linux and one for mac.